optimum/commands/export/onnx.py

```diff
@@ -217,6 +217,11 @@ def parse_args_onnx(parser):
         default=DEFAULT_DUMMY_SHAPES["nb_points_per_image"],
         help="For Segment Anything. It corresponds to the number of points per segmentation masks.",
     )
+    optional_group.add_argument(
+        "--legacy",
```

Comment:
We could merge the `no_position_ids` and `legacy` as they correspond to the previous export behavior and `no_position_ids` is not in a release yet. WDYT?

Reply:
Yes good idea, will merge both